### PR TITLE
Feature#20 my complaints screen design

### LINKED ISSUE #17: feature#20 My Complaints Screen Design
## Description
We need to implement <My Complaints> section screen design, this feature includes all necessary components, styles, files and dependencies. Our goal is to complete an app View that will allow the user to navigate and interact through this additional section.

This section should show the user a list of their submitted complaints, as well as have an option to edit and cancel complaints. The list should be ordered by date.

## Provisional Design
image

## Acceptance Criteria
- [x] Have a list of user complaints
  - [x] single complaint card component
- [x] Complaints ordered by date
- [x] Edit Button for every complaint
- [x] Cancel Button for every complaint

### PR DESCRIPTION
**Pull request by francisco-sle**
## Description
I've coded several components to show My Complaints section, result of issue #17, the goal was to create a screen that allows the user to see their submitted complaints in a list, and providing buttons for edition, deletion and information.
There is no functionality related with Firebase, User Session, Data fetching and Redux yet. However, this screen enables future implementations.
Additional information such as formatted dates and sorted arrays are managed in Methods folder, this was made to keep file structure intention.

## Files to review
- src/containers/MyComplaints/layout/index.js
- src/containers/MyComplaints/styles/index.js
- src/library/methods/dateFormatter.js
- src/library/methods/sortArrayByDate.js
- src/library/methods/normalize.js
- src/containers/MyComplaints/dummyData/index.js 

## Installed packages

- [Moment.js](https://momentjs.com/)
- [React Navigation](https://reactnavigation.org/docs/getting-started/)
- [react-native-vector-icons](https://github.com/oblador/react-native-vector-icons)

### CODE REVIEW

File: src/containers/MyComplaints/dummyData/index.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.',
+  date: 'Ocurrido el 6 mayo, 2021',
+  location:
+    'Hacienda Los Fresnos, Av. Los Fresnos #345, Jalisco, Tlatomulco de Zúñiga',
+  author: 'Denuncia por José Támara',
+  authorDate: '7 de mayo, 2021',
+  fav: false,
+};
+
+let MyCompEmpty = [];
```
Comment (fernandoamz) [inline src/containers/MyComplaints/dummyData/index.js]: The variable's name should be complete in order to give a clear idea about what the function does

Example:
```suggestion
let MyCompanyEmpty = [];
```

File: src/containers/MyComplaints/layout/index.js [hunk 2]
```diff
@@ -1,18 +1,117 @@
...
+import {DATA, MyComp, MyCompEmpty} from '../dummyData';
 
-const SendDataAdd = props => {
-  const next = () => {
-    props.navigation.navigate('TitleDescriptionAdd', {
-      ...props.route.params,
-    });
+const ComplaintItem = ({category, title, date, id}) => {
+  const onEditPress = () => {
+    //Edit View
```
Comment (fernandoamz) [inline src/containers/MyComplaints/layout/index.js]: Remove all the comments that you don't need.

File: src/containers/MyComplaints/layout/index.js [hunk 3]
```diff
@@ -1,18 +1,117 @@
...
+        },
+        {
+          text: 'Sí, borrar',
+          onPress: () => {
+            //delete Complaint functionality
+          },
+        },
+      ],
+    );
+  };
```
Comment (fernandoamz) [inline src/containers/MyComplaints/layout/index.js]: Move all the text in a constants file in order to internalization be easier to apply

**Review by fernandoamz — CHANGES_REQUESTED** [review]

**Review by YilKanda — DISMISSED** [review]
LGTM!

File: src/containers/MyComplaints/layout/index.js (hunk 2 above)
Comment (francisco-sle) [reply src/containers/MyComplaints/layout/index.js]: I've removed all unnecessary comments, as hooks and variable names already tell the purpose of the code.
Thanks

File: src/containers/MyComplaints/layout/index.js (hunk 3 above)
Comment (francisco-sle) [reply src/containers/MyComplaints/layout/index.js]: Done, this sort of messages are now retrieved from a different file located at library/constant directory.

File: src/containers/MyComplaints/dummyData/index.js (hunk 1 above)
Comment (francisco-sle) [reply src/containers/MyComplaints/dummyData/index.js]: Fixed!

**Comment by francisco-sle** [comment]
> LGTM!

@YilKanda May I ask for another code approval? it seem that I accidentally dismissed your review due to new commits I made to solve requested changes from another reviewer.

Thanks in advance.

**Review by jkzasori — APPROVED** [review]
LGTM!

**Review by ccortes15 — APPROVED** [review]
LGTM!

**Pull request merged**